docs/app-directory/overview.md

```diff
@@ -7,18 +7,75 @@ hide_title: true
+
+
+#### Apps are discoverable
+
+For large institutions, it can be difficult to keep track of all the apps (developed both in-house and by vendors), since a typical desktop could have many. Users can search appD to discover the apps they need. An app might already reside on their system or be available to them over the internet, but if they don’t have a way to search the apps available to them, they won’t be able to find it. An appD provides a way for users to discover the apps they need.
+
+An appD makes it possible to discover info about apps that reside on various domains, not just the one domain the appD itself is hosted on. In addition, you can find details about how to launch the apps in multiple, diverse environments. This is in sharp contrast to the [W3C's Web App Manifest](https://www.w3.org/TR/appmanifest/), which is hosted on the same domain as the app, is limited to web apps, and is generally used to 'install' an app that the user has already accessed.
+
+
+#### Updating apps
+
+Typically, software evolves over time. The app versions you are running today will not be the same ones you need tomorrow. Therefore, you will need to upgrade apps periodically. Very few people look forward to upgrading, but appD and web deployment can make it easier for you. To roll out a new version of your app, either update the existing entry for it in appD or add a new entry for that version (allowing users to select the version they will use).
+
+
+#### Agent-agnostic
+
+As a part of the FDC3 standard, appD isn't tied to any specific vendor. This is important, as it allows you more flexibility in that you are not tied to any specific container or desktop agent implementation. If at any point you want to switch to a different desktop agent, the process won’t be prohibitively difficult. The existing appD will work without any additional effort from you, as long as your new desktop agent is also FDC3-compliant. This is in contrast with proprietary solutions, where you would have to reconstruct the entire ecosystem from scratch. 
+
+AppD reduces fragmentation in the market and allows end-users more flexibility in what applications can be included in their desktop.
+
+
+#### Intent resolution
+
+AppD provides information to the Desktop Agent on which applications support particular [intents](../intents/overview) and the context types they require as input to them. This allows the Desktop Agent to implement an [intent resolver](../api/spec#resolvers) that can launch applications and pass the intent and context to them to operate on, supporting workflows between applications that didn't require prior bilateral agreements between the application providers.
```

Comment:
I wouldn't say that apps "require" input - they "work with", "handle" or "understand"

Reply:
They do in fact require the context type to match what they are configured to receive or `findIntent` (when context is supplied) is not supposed to return an app as an option and `raiseIntent` (where context is always passed) shouldn't cause it to show up as an option in a resolver UI... 

If it feels better to write 'need' I can swap it out, but the meaning is the same. To use 'support' I'd need to replace the first use of it in that sentence, e.g.

```suggestion
AppD provides information to the Desktop Agent on which applications can handle particular [intents](../intents/overview) and the context types they support as input to them. This allows the Desktop Agent to implement an [intent resolver](../api/spec#resolvers) that can launch applications and pass the intent and context to them to operate on, supporting workflows between applications that didn't require prior bilateral agreements between the application providers.
```

Reply:
Feels much better!